lib/chef_zero/chef_data/data_normalizer.rb

```diff
@@ -92,24 +93,29 @@ def self.normalize_cookbook(endpoint, org_prefix, cookbook, name, version, base_
             end
           end
           cookbook['name'] ||= "#{name}-#{version}"
-          # TODO this feels wrong, but the real chef server doesn't expand this default
-    #      cookbook['version'] ||= version
-          cookbook['cookbook_name'] ||= name
+          # TODO it feels wrong, but the real chef server doesn't expand 'version', so we don't either.
+
           cookbook['frozen?'] ||= false
           cookbook['metadata'] ||= {}
           cookbook['metadata']['version'] ||= version
-          # Sad to not be expanding defaults just because Chef doesn't :(
-  #        cookbook['metadata']['name'] ||= name
-  #        cookbook['metadata']['description'] ||= "A fabulous new cookbook"
+
+          # defaults set by the client and not the Server:
+          # metadata[name, description, maintainer, maintainer_email, license]
+
           cookbook['metadata']['long_description'] ||= ""
-  #        cookbook['metadata']['maintainer'] ||= "YOUR_COMPANY_NAME"
-  #        cookbook['metadata']['maintainer_email'] ||= "YOUR_EMAIL"
-  #        cookbook['metadata']['license'] ||= "none"
           cookbook['metadata']['dependencies'] ||= {}
           cookbook['metadata']['attributes'] ||= {}
           cookbook['metadata']['recipes'] ||= {}
         end
-        cookbook['json_class'] ||= 'Chef::CookbookVersion'
+
+        # not sure if there is a material reason for these differences.
```

Comment:
It's because when `json_class` is present, Chef's default API client code will try to convert the JSON to a custom ruby object. I didn't want that behavior when dealing with cookbook artifact objects.